src/components/Navbars.tsx

```diff
@@ -39,13 +27,11 @@ import AutosaveIndicator, {
 } from './AutosaveIndicator';
 import AutosaveContext from 'AutosaveContext';
 import Sidebar from './Sidebar';
-import { formatUserName } from 'shared/strings';
-import { signOut } from "next-auth/react";
 
 export const sidebarWidth = 60;
-export const topbarHeight = "60px";
+export const topbarHeight = "0px";
```

Comment:
Now that topbarHeight and sidebarContentMarginTop are no longer useful, remove them and their references completely.